errors/abci_test.go

```diff
@@ -202,20 +199,15 @@ func TestABCIInfoSerializeErr(t *testing.T) {
 		},
 		"multiErr default encoder": {
 			src: Append(myErrMsg, myErrState),
-			exp: `{"data":[{"code":4,"log":"test: invalid message"},{"code":10,"log":"test: invalid state"}]}`,
+			exp: Append(myErrMsg, myErrState).Error(),
 		},
 		"multiErr default with internal": {
 			src: Append(myErrMsg, myPanic),
-			exp: `{"data":[{"code":4,"log":"test: invalid message"},{"code":111222,"log":"internal error"}]}`,
-		},
-		"multiErr debug": {
-			src:   Append(myErrMsg, myErrState),
-			debug: true,
-			exp:   fmt.Sprintf(`{"data":[{"code":4,"log":%s},{"code":10,"log":%s}]}`, string(myErrMsgSTJson), string(myErrStateSTJson)),
+			exp: "internal error",
 		},
 		"redact in default encoder": {
 			src: myPanic,
-			exp: internalABCILog,
+			exp: "internal error",
```

Comment:
More tests added in https://github.com/iov-one/weave/pull/776/commits/7f383435328cc6469eeb371e402e4eb56509f7b3

There is one test case that was not tested because it is not logical what should happen.
When formatting a multi error to show stack trace, which stack trace should be displayed, assuming that all errors in the collection have one? For example
```go
err = Append(
    Wrap(ErrNotFound, "a"),
    Wrap(Append(Wrap(ErrEmpty, "b"), Wrap(ErrState, "c")), "d")
```
The current implementation returns the first stack trace, in this case `"a"`.